utils/utils.go

```diff
@@ -46,15 +48,31 @@ func XcodeBuildVersion() (XcodebuildVersionModel, error) {
 }
 
 // IsToolInstalled ...
-func IsToolInstalled(name string) bool {
-	cmd := command.New("which", name)
-	out, err := cmd.RunAndReturnTrimmedCombinedOutput()
-	return err == nil && out != ""
+func IsToolInstalled(name, version string) (bool, error) {
+	return rubycommand.IsGemInstalled(name, version)
 }
 
 // IsXcprettyInstalled ...
-func IsXcprettyInstalled() bool {
-	return IsToolInstalled("xcpretty")
+func IsXcprettyInstalled() (bool, error) {
+	return IsToolInstalled("xcpretty", "")
+}
+
+// InstallXcpretty ...
+func InstallXcpretty() error {
+	cmds, err := rubycommand.GemInstall("xcpretty", "")
+	if err != nil {
+		return fmt.Errorf("failed to create command model, error: %s", err)
+	}
+
+	for _, cmd := range cmds {
+		log.Donef("$ %s", cmd.PrintableCommandArgs())
```

Comment:
do not use green coloring, this is just a small utility, but not a big milestone in the step, like other Done logs